### PR TITLE
2 packages from sapristi/easy_logging at 0.5

### PR DESCRIPTION
**Pull request by sapristi**
This pull-request concerns:
-`easy_logging.0.5`: Module to log messages. Aimed at being both powerful and easy to use
-`easy_logging_yojson.0.5`: Configuration loader for easy_logging with yojson backend

---
* Homepage: https://sapristi.github.io/easy_logging/
* Source repo: git+https://github.com/sapristi/easy_logging.git
* Bug tracker: https://github.com/sapristi/easy_logging/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/easy_logging/easy_logging.0.5/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+maintainer: "mathiasmillet@gmail.com"
+authors: "Mathias Millet"
+license: "GPL"
+homepage: "https://sapristi.github.io/easy_logging/"
+bug-reports: "https://github.com/sapristi/easy_logging/issues"
+dev-repo: "git+https://github.com/sapristi/easy_logging.git"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build & >= "1.0"}
```
Comment (kit-ty-kate) [inline packages/easy_logging/easy_logging.0.5/opam]:
```suggestion
  "dune" {build & >= "1.6.3"}
```
According to the `dune-project` file, this package and the other one require dune >= 1.6.3

**Comment by kit-ty-kate** [comment]
The checksum doesn't seem to match:
```
[ERROR] The sources of the following couldn't be obtained, aborting:
          - easy_logging.0.5:
            https://github.com/sapristi/easy_logging/tarball/v0.5 (Bad checksum, expected md5=931cce524bf8a9815e15edebf6f4d99a)
```

File: packages/easy_logging/easy_logging.0.5/opam (hunk 1 above)
Comment (sapristi) [reply packages/easy_logging/easy_logging.0.5/opam]: Thank you @kit-ty-kate.

File: packages/easy_logging/easy_logging.0.5/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+maintainer: "mathiasmillet@gmail.com"
+authors: "Mathias Millet"
+license: "GPL"
+homepage: "https://sapristi.github.io/easy_logging/"
+bug-reports: "https://github.com/sapristi/easy_logging/issues"
+dev-repo: "git+https://github.com/sapristi/easy_logging.git"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build & >= "1.8"}
```
Comment (kit-ty-kate) [inline packages/easy_logging/easy_logging.0.5/opam]:
```suggestion
  "dune" {>= "1.8"}
```
Dune should not be flagged as a build dependency anymore (see #14266)

File: packages/easy_logging_yojson/easy_logging_yojson.0.5/opam [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+maintainer: "mathiasmillet@gmail.com"
+authors: "Mathias Millet"
+license: "GPL"
+homepage: "https://sapristi.github.io/easy_logging/"
+bug-reports: "https://github.com/sapristi/easy_logging/issues"
+dev-repo: "git+https://github.com/sapristi/easy_logging.git"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build & >= "1.8"}
```
Comment (kit-ty-kate) [inline packages/easy_logging_yojson/easy_logging_yojson.0.5/opam]:
```suggestion
  "dune" {>= "1.8"}
```

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**